modules/psap-special-resource-operator-installing.adoc

```diff
@@ -0,0 +1,171 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="installing-the-special-resource-operator_{context}"]
+= Installing the Special Resource Operator
+
+As a cluster administrator, you can install the Special Resource Operator (SRO) using the {product-title} CLI or the web console.
+
+[id="install-operator-cli_{context}"]
+== Installing the Special Resource Operator using the CLI
+
+As a cluster administrator, you can install SRO using the CLI.
+
+.Prerequisites
+
+* You have a running {product-title} cluster
```

Comment:
s/cluster/cluster./

Reply:
Please add punctuation to the end of bullet point sentences.

Reply:
Fixed, thanks